tests/test_pipreqs.py

```diff
@@ -82,6 +83,14 @@ def setUpClass(cls):
         cls.requirements_path = os.path.join(cls.project, "requirements.txt")
         cls.alt_requirement_path = os.path.join(cls.project, "requirements2.txt")
         cls.non_existing_filepath = "xpto"
+        
+        cls.project_with_notebooks = os.path.join(os.path.dirname(__file__), "_data_notebook")
+        cls.project_with_invalid_notebooks = os.path.join(os.path.dirname(__file__), "_invalid_data_notebook")
+        cls.compatible_files = {
```

Comment:
maybe rename compatible_files to something more intuitive? maybe its just me, but this name makes no sense haha

Reply:
maybe something like equivalent? with keys notebook and  python (or .py)

Reply:
indeed, it is not so clear about its purpose. we just changed it to a clearer name.